bindings/java/README.md

```diff
@@ -62,14 +62,48 @@ You can use Maven to build both Rust dynamic lib and JAR files with one command
 ./mvnw clean package -DskipTests=true
 ```
 
+## Setup Tests
```

Comment:
I have an idea to by default opt out service tests with Jupiter's tags. These are advanced topics so you'd better talk about them later instead of first, or even talk into details in a new doc file.